README.md

```diff
@@ -131,6 +131,7 @@ When using CMake to compile PhASAR the following optional parameters can be used
 | <b>BUILD_SHARED_LIBS</b> : BOOL | Build shared libraries (default is ON) |
 | <b>CMAKE_BUILD_TYPE</b> : STRING | Build PhASAR in 'Debug' or 'Release' mode <br> (default is 'Debug') |
 | <b>CMAKE_INSTALL_PREFIX</b> : PATH | Path where PhASAR will be installed if <br> “make install” is invoked or the “install” <br> target is built (default is /usr/local/phasar) |
+| <b>PHASAR_CONFIG_INSTALL_PREFIX</b> : PATH | Path where PhASAR's configuration files will be installed if <br> “make install” is invoked or the “install” <br> target is built. Expression will be evaluated within CMAKE_INSTALL_PREFIX, so prefer absolute paths (default is $(HOME)/.config/) |
```

Comment:
make install or ninja install - how do we generalize this without bloating the text?

Reply:
In #559 `make` is already replaced by `ninja`. Maybe, we can just adapt this accordingy when merging